From 00b409fafa15cee022849381fd3559e2a3539011 Mon Sep 17 00:00:00 2001
From: "Munoz, Obed N" <obed.n.munoz@intel.com>
Date: Fri, 28 Aug 2015 17:38:20 -0500
Subject: [PATCH] Remove SELECT 1

Signed-off-by: Munoz, Obed N <obed.n.munoz@intel.com>
---
 oslo_db/sqlalchemy/session.py | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/oslo_db/sqlalchemy/session.py b/oslo_db/sqlalchemy/session.py
index 6a0355d..75f25bd 100644
--- a/oslo_db/sqlalchemy/session.py
+++ b/oslo_db/sqlalchemy/session.py
@@ -333,7 +333,8 @@ def _connect_ping_listener(connection, branch):
     try:
         # run a SELECT 1.   use a core select() so that
         # any details like that needed by Oracle, DB2 etc. are handled.
-        connection.scalar(select([1]))
+        # connection.scalar(select([1]))
+        pass
     except exception.DBConnectionError:
         # catch DBConnectionError, which is raised by the filter
         # system.
@@ -341,7 +342,8 @@ def _connect_ping_listener(connection, branch):
         # "invalid", but the pool should be ready to return
         # new connections assuming they are good now.
         # run the select again to re-validate the Connection.
-        connection.scalar(select([1]))
+        # connection.scalar(select([1]))
+        pass
     finally:
         connection.should_close_with_result = save_should_close_with_result
 
-- 
2.5.0

